javascript/app.js

```diff
@@ -0,0 +1,481 @@
+//4
+ourClass.push("Cloud City");
+console.log(ourClass[5]);
+
+//E.Mix it up
+
+//1
+const myArray = [5, 10, 500, 20];
+myArray.push("Egon", "20");
+console.log(myArray[4]);
+//2 
+myArray.shift();
+//3 
+myArray.unshift("Bob Marley");
+console.log(myArray[0]);
+//4 
+myArray.splice(4, 5);
+console.log(myArray);
+//5 Yes this does mututate the array. It takes the elements, reverses them, creating a new array. It does not apply the values to the original array.
+myArray.reverse()
+console.log(myArray);
+
+//F Biggie Smalls
+
+const number = 105;
+if (number < 100) {
+	console.log("Little number");
+} else {
+	console.log("Big number");
+}
+
+//G Monkey in the middle
+const monkeyNumber = 6;
+
+if (monkeyNumber < 5) {
+	console.log("Little number");
+} else if (monkeyNumber > 10) {
+	console.log("Big Number");
+} else {
+	console.log("Monkey");
+}
+
+//H What's in your closet?
+const kristynsCloset = [
+  "left shoe",
+  "cowboy boots",
+  "right sock",
+  "GA hoodie",
+  "green pants",
+  "yellow knit hat",
+  "marshmallow peeps"
+];
+
+// Thom's closet is more complicated. Check out this nested data structure!!
+const thomsCloset = [
+  [
+    // These are Thom's shirts
+    "grey button-up",
+    "dark grey button-up",
+    "light blue button-up",
+    "blue button-up",
+  ],[
+    // These are Thom's pants
+    "grey jeans",
+    "jeans",
+    "PJs"
+  ],[
+    // Thom's accessories
+    "wool mittens",
+    "wool scarf",
+    "raybans"
+  ]
+];
+
+//1
+console.log(`Kristyn is rocking that ${kristynsCloset[2]} today!`);
+//2 
+kristynsCloset.splice(6, 0, "raybans");
+//start at index 6, remove nothing 0, add "raybans"
+console.log(kristynsCloset);
+//3 
+kristynsCloset[6] = "stained knit hat";
+console.log(kristynsCloset);
+//4, 5, 6, 7
+console.log(`Thom is looking fierce in a ${thomsCloset[0][2]}, ${thomsCloset[1][0]} and ${thomsCloset[2][2]}!`);
+//8 
+thomsCloset[1][2] = "Footie Pajamas";
+
+
+//IV Functions
+
+//A printGreeting
+//done it... 
+
+//B printCool
+const printCool = (name) => {
+	console.log(`${name} is cool`);
+}
+printCool('Ashley');
+
+//C calculateCube
+//at first I put console log outside the loop, but then I fixed that. 
+const calculateCube = (number) => {
+	console.log(Math.pow(number, 3));
+};
+calculateCube(6);
+
+
+//D isVowel
+
+const isVowel = (character) => {
+	const vowels = ['a', 'e', 'i', 'o', 'u'];
+	if (character == vowels) {
+		console.log(character);
+		return true;
+	} else {
+		console.log(false);
+		return false;
+	}
+}
+isVowel('t');
+
+//E getTwoLengths
+
+const getTwoLengths = (string1, string2) => {
+	thisArray = [];
+	thisArray.push(string1.length);
+	thisArray.push(string2.length);
+	console.log(thisArray);
+}
+getTwoLengths("Ashley", "Fueger");
+
+//F getMultipleLengths
+
+const getMultipleLengths = (arrayString) => {
+	const familyArray = [];
+for (let i = 0; i < arrayString.length; i++) {
+	familyArray.push(arrayString[i].length);
+	}
+	console.log(familyArray);
+	return familyArray;
+} 
```

Comment:
fix the indentation.  paste this code into beautifier.io and then hand-edit the indentation in your answer to match the restuls